### ISSUE TITLE
Purification moves seem to be missing

### ISSUE THREAD

**Issue by chaseadams509** [PokeAPI/pokedex#21]
I was checking some moves, and noticed that it seems that purification moves are not listed. I checked Claydol with refresh, and Moltres. Neither had their purification moves listed. And it seems to be intended to be supported because there is a move learn method for "xd-purification", /move-learn-method/9. (I'm not sure what move-learn-method/7 is however).

**Comment by Naramsim** [comment]
Hi @chaseadams509,

thanks for spotting it. Is purification a single move or a set of moves? Do you know if these ones were added in gen-8?

**Comment by chaseadams509** [comment]
No, all come from Pokemon XD: Gale of Darkness. Most only have 1 move, but a few have 2 or 3. This seems to be a complete list: https://serebii.net/xd/pokemon-attacks.shtml

**Comment by Naramsim** [comment]
Ok, let me understand it. So purification moves aren't new moves. For example, Beedrill can come to learn Baton Pass.

So in my opinion we already have the moves in our database, but we lack the mechanism to tell which Pokemon can learn a move through purification.

Am I correct?

**Comment by chaseadams509** [comment]
They aren't new moves, but a method for learning moves that they may not normally learn (similar to how some pokemon only learn moves via move tutors). The one that I noticed was Claydol can learn Refresh only through XD purification. So when I was querying pokemon/claydol and looked at the list of moves, refresh did not appear. However, Claydol can learn Refresh in XD via purification. It seems that the mechanism was added (i.e. move-learn-method/9), but not updated to include them.

So, if I query Claydol, I should expect an entry under moves:

```
    {
      "move": {
        "name": "refresh",
        "url": "https://pokeapi.co/api/v2/move/287/"
      },
      "version_group_details": [
        {
          "level_learned_at": 0,
          "move_learn_method": {
            "name": "xd-purification",
            "url": "https://pokeapi.co/api/v2/move-learn-method/9/"
          },
          "version_group": {
            "name": "xd",
            "url": "https://pokeapi.co/api/v2/version-group/13/"
          }
        }
      ]
    },
```

**Comment by chaseadams509** [comment]
I think relatedly, there are shadow moves that I see in. your database (e.g. move/1003 i.e. shadow-blitz) but not attached to any pokemon (e.g. baltoy). This would have move-learn-method/8, such as:
```
    {
      "move": {
        "name": "shadow-blitz",
        "url": "https://pokeapi.co/api/v2/move/1003/"
      },
      "version_group_details": [
        {
          "level_learned_at": 0,
          "move_learn_method": {
            "name": "xd-shadow",
            "url": "https://pokeapi.co/api/v2/move-learn-method/8/"
          },
          "version_group": {
            "name": "xd",
            "url": "https://pokeapi.co/api/v2/version-group/13/"
          }
        }
      ]
    },
```

**Comment by Naramsim** [comment]
We need to scrape this kind of data then. A volunteer would be much appreciated.
